tests/testthat/test_correctness_pnbd.R

```diff
@@ -49,4 +49,102 @@ fct.testthat.correctness.dyncov(data.apparelTrans=apparelTrans, data.apparelDynC
 
 # ** Static cov??
 
+context("Correctness - PNBD nocov - Expectation")
 
+test_that("Expectation in Rcpp matches expectation in R (nocov)", {
+
+  # No cov ---------------------------------------------------------------------------------------------------
+  expect_silent(clv.cdnow <- clvdata(data.transactions = cdnow,
+                                     date.format = "ymd", time.unit = "W", estimation.split = 38,
+                                     name.id = "Id", name.date = "Date", name.price = "Price"))
+
+
+  expect_silent(obj.fitted <- do.call(pnbd, list(clv.data = clv.cdnow, verbose = FALSE)))
+
+
+  params_i <- obj.fitted@cbs[, c("Id", "T.cal", "date.first.actual.trans")]
```

Comment:
This seems very repetitive, same for pnbd and bgnbd and nearly same for cov and nocov. Could that not be put in a separate function in one of the "helper_*" files (has to be in "helper_" file) and then re-used?